nexus-sdk/build.gradle

```diff
@@ -30,3 +31,62 @@ spotless {
 }
 
 classes.dependsOn 'spotlessApply'
+
+ext.getVersionName = { ->
+    def version = System.getenv('NEXUS_SDK_TAG')
```

Comment:
There is not really yet any value to the more advanced Temporal Java SDK approach of temporarily tagging and then `git describe --tags` to pull it right out. So our publishing step requires this env var instead.